soccer/gameplay/plays/restarts/placement.py

```diff
@@ -0,0 +1,50 @@
+import standard_play
+import behavior
+import robocup
+import main
+import constants
+import tactics.line_up
+import tactics.our_placement
+
+
+# one robot places the ball, the others just line up and wait
+class Placement(standard_play.StandardPlay):
+    def __init__(self):
+        super().__init__(continuous=True)
+
+        self.remove_subbehavior('defense')
```

Comment:
If the placement is set to be near our goal, the goalie will come out and kick the ball across the field. This is detrimental if its our placement and I think breaks the rules if it is theirs.